lib/commands/geolocation.js

```diff
@@ -26,6 +26,12 @@ const GPS_COORDINATES_PATTERN = /data="(-?[\d.]+)\s+(-?[\d.]+)\s+(-?[\d.]+)"/;
  * @property {?number|string} [satellites=12] - Number of satellites being tracked (1-12).
  * This value is ignored on real devices.
  * @property {?number|string} [speed] - Valid speed value.
+ * https://developer.android.com/reference/android/location/Location#setSpeed(float)
+ * @property {?number|string} [bearing] - Valid bearing value.
```

Comment:
it looks like jsdoc is not happy about such notation (with prefixed question mark). Instead it should look like

`@property {number|string} [bearing]` or `@property {number|string|null} [bearing]` if we would like to also add null values in addition to undefined